### PR TITLE
Added 1 file and Updated 1 file

### LINKED ISSUE #15: No String resource of any Component.
No component of any activity/screen had a @string resource.
Also in the landscape mode the button 2 shifts a little left.

### PR DESCRIPTION
**Pull request by parjanyaacoder**
All the components of all screen were not having a @string resource.Added a @string resource to every component.
In the landscape orientation the button 2 was getting shifted a little left.Corrected this also.

Fixes #15

### CODE REVIEW

**Comment by somenath1435** [comment]
@parjanyaacoder please define colors in colors.xml

**Comment by parjanyaacoder** [comment]
I did not get you what colors which colors

File: activity_main.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.395"
+        android:contentDescription="@string/todo" />
+
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/robotalk"
+        android:textColor="#000000"
```
Comment (somenath1435) [inline activity_main.xml:28]: add color in colors.xml

**Review by somenath1435 — CHANGES_REQUESTED** [review]

File: activity_main.xml (hunk 1 above)
Comment (parjanyaacoder) [reply activity_main.xml:28]: So I can add any color of my choice or a particular color

Comment (somenath1435) [reply activity_main.xml:28]: #00000 stands for white, you should add that in colors.xml(predefined file) inside of values folder adjacent to strings.xml

Comment (parjanyaacoder) [reply activity_main.xml:28]: I checked your code 
Files like activity_text_to_speech and activity_speech_to_text also contains the code 
android:textColor="#value" So do you want me to add all these values from different xml files to the color.xml file

Comment (somenath1435) [reply activity_main.xml:28]: Absolutely, if needed,create a new issue and solve

Comment (parjanyaacoder) [reply activity_main.xml:28]: I think I have done everything as per your guidance.Now if you find it right then please merge my pull request.

Comment (somenath1435) [reply activity_main.xml:28]: Last thing, please attach a screenshot for landscape mode(both old and new), so that difference is visible

**Pull request merged**